projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.QuoteTemplates.php

```diff
@@ -1070,7 +1070,7 @@ private function tidy_quotetemplate($obj=false,$withCustomFields=false){
             $res['owner'] = $obj->zbs_owner;
 
 
-			$res['title']            = $this->stripSlashes( $obj->zbsqt_title );
+			$res['title']            = wp_kses( html_entity_decode( $obj->zbsqt_title, ENT_QUOTES, 'UTF-8' ), $zbs->acceptable_restricted_html );
```

Comment:
🤔 Why are we converting to HTML entities at any point? Are we saving things wrong?

Reply:
Here we're making the same change as on line 1079 below for notes, which fixed the notes rendering issue (see https://github.com/Automattic/jetpack/pull/33614) and also fixes the title escaping issue. You made a suggestion in that PR regarding using a textarea and using sanitize_textarea_field. That could also be a good idea here. Possibly for a follow-up in that case 🤔

Reply:
Yeah, for notes it's a textarea, so we'd expect HTML, but for a title we'd expect regular text as-is. But out of scope here.